src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -306,6 +336,23 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     }
   }
 
+  private handleQuerySuggestSelected(args: IQuerySuggestSelected) {
+    this.executeQueryHover(args.suggestion);
+  }
+
+  private executeQueryHover(suggestion: string) {
+    let previousQueryOptions = this.queryController.getLastQuery();
+    previousQueryOptions.q = suggestion;
+    previousQueryOptions.numberOfResults = this.options.numberOfPreviewResult;
+    this.queryController
+      .getEndpoint()
+      .search(previousQueryOptions)
+      .then(results => {
+        //I now have the results for the preview.
```

Comment:
Should there be some logic here? or will that be in a future PR?

Reply:
in a future pr.
I'll remove it for this pr.